recipes/zephyr-7b-beta/sft/config_lora.yaml

```diff
@@ -37,8 +37,9 @@ log_level: info
 logging_steps: 5  
 logging_strategy: steps
 lr_scheduler_type: cosine
+warmup_ratio: 0.1
 max_seq_length: 2048
-max_steps: -1
+max_steps: 272
```

Comment:
I think `max_steps=-1` because `num_train_epochs` is used instead

Reply:
Yeah, but the ConstantLengthDataset doesn't know how many steps it will run, so the scheduler can't setup the warmup cycle correctly

Reply:
I amy try to fix this in trl